### DISCUSSION TITLE
graph.on('graph:mouseup', (e) => {});事件无法触发

### DISCUSSION THREAD

**Discussion by ouxiaowei** [antvis/X6#2384] [Q&A] [answered]
1.是否有这个事件；
2.如果没有，后期是否能支持；
![image](https://user-images.githubusercontent.com/37791464/179947035-c6ca2b28-17da-4315-bac3-349b1780b499.png)

**Comment by NewByVector** [comment, marked as answer]
有 `blank:mouseup` 这个事件。

**Reply by ouxiaowei** [reply]
烦请问下，blank:mouseup怎么绑定到graph上呢？

**Reply by NewByVector** [reply]
```ts
graph.on('blank:mouseup', () => {})
```

**Reply by ouxiaowei** [reply]
我试了下，'blank:mouseup'只支持在画布上的按下才能获取的松开事件，如果是从画布外的其他dom节点按下再进入画布松开，就不触发。画布外进入这样的情况有提供事件监听吗？

**Reply by NewByVector** [reply]
不支持这样的，如果要实现你这个功能，可以直接监听 body 上的事件。
